### DISCUSSION TITLE
validator sorting

### DISCUSSION THREAD

**Discussion by AutoStake1** [phantom/docs#153] [General]
Hey

We  were wondering how the default sorting of validators works on phantom, how can a validator get on the default list displayed?

Our validator only appears after starting to type into the search bar - why is this?

**Comment by adamdelphantom** [comment]
Hello @AutoStake-com,

Validators on the Phantom staking list is sorted by the total amount of rewards given out to stakers.

**Comment by AutoStake1** [comment]
Ah, I see thank you for the clarification!

So it forever favours the oldest validators on the network?
Or is there any way we can catch up?

**Reply by adamdelphantom** [reply]
Welcome! The more rewards you provide to stakers over time the better chance you have of catching up.

**Reply by AutoStake1** [reply]
Thanks!

**Comment by AutoStake1** [comment]
Barring any catastrophic event happening to all the oldest validators, it is impossible for newer validators to ever catch, hurting the decentralisation of the network.

Might i suggest sorting by predicted APR instead? this way many validators can take it in turns at the top of the list for even stake distribution

**Reply by adamdelphantom** [reply]
@AutoStake-com Thank you for the feedback and suggestion! I would suggest submitting this as a feature request on our canny board. https://phantom.canny.io

**Reply by AutoStake1** [reply]
Thanks
